src/test/java/org/example/tokpik_be/tag/controller/TopicTagControllerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.example.tokpik_be.tag.controller;
+
+import static org.mockito.BDDMockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.List;
+
+import org.example.tokpik_be.exception.GeneralException;
+import org.example.tokpik_be.exception.UserException;
+import org.example.tokpik_be.support.ControllerTestSupport;
+import org.example.tokpik_be.tag.controller.TopicTagController;
+import org.example.tokpik_be.tag.dto.response.UserTopicTagResponse;
+import org.example.tokpik_be.tag.service.TopicTagService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.web.servlet.ResultActions;
+
+public class TopicTagControllerTest extends ControllerTestSupport {
+    @Mock
+    private TopicTagService topicTagService;
+
+    @InjectMocks
+    private TopicTagController topicTagController;
+
+    @Override
+    protected Object initController() {
+
+        return topicTagController;
+    }
+
+    @Nested
+    @DisplayName("사용자 대화 태그 조회 시")
```

Comment:
```suggestion
    @DisplayName("사용자 대화 태그 조회 시 ")
```
사소한 부분이지만 공백 하나만 부탁 드려요~ 🙏🏻